etc/kubernetes/manifests/overlays/local/keycloak/deployment.yaml

```diff
@@ -26,8 +26,8 @@ spec:
               value: "admin"
             - name: JAVA_OPTS_APPEND
               value: "-Dkeycloak.import=/opt/keycloak/data/import/rhoas-realm.json,/opt/keycloak/data/import/rhoas-kafka-sre-realm.json"
-            - name: KC_HOSTNAME
-              value: "cos-fleet-manager-kc"
+            - name: KC_PROXY
+              value: "edge"
```

Comment:
@lburgazzoli if I remove the route, I'll probably also have to revert this. Makes me wonder whether I should just move all the changes to the qe overlay, including the user script and job, and just patch the keycloak deployment there.

Reply:
I think for the moment  yes, that is probably the best option. 
We should probably explore if we can do the same locally by using an ingress